### PR TITLE
Add Requests python package to resolwebio/utils Docker image

### PR DESCRIPTION
**Pull request by hadalin**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ Added
   subtype processes have the input and output fields that are defined in the
   abstract process. For users, it is much more clear and predictable if they
   can expect the same set of outputs on all processes of the same type.
+* ``requests`` Python package to ``resolwebio/utils`` Docker image
```
Comment (mzganec) [inline docs/CHANGELOG.rst]: Prepend `Add`. You will need also `Trimmomatic` to do quality control.

Comment (mstajdohar) [reply docs/CHANGELOG.rst]: I think quality control is now in another process.

Comment (hadalin) [reply docs/CHANGELOG.rst]: >Prepend Add

Isn't this redundant information? It's under _Added_ section. Does @tjanez approve?

Comment (hadalin) [reply docs/CHANGELOG.rst]: >I think quality control is now in another process.

This image is used for `data:basespace`. @mzganec will update I think `rnaseq` which will be used for `basespace-fastq-*`.

**Comment by mzganec** [comment]
We regularly use _Add_ in the change log, that's why I suggested it. Otherwise I don't mind.

Locally, the image builds correctly.

File: docs/CHANGELOG.rst (hunk 1 above)
Comment (tjanez) [reply docs/CHANGELOG.rst]: > >Prepend Add

> Isn't this redundant information? It's under _Added_ section. Does @tjanez approve?

I agree with @mzganec:

1. It is strange to start a CHANGELOG entry with some fixed-spaced identifier such as ``requests``.

2. CHANGELOG is not a work of art, so a bit of redundancy, IMHO, can be outweighed by simplicity of using the same sentence as in the commit message.

**Comment by tjanez** [comment]
My opinion added in the [comment thread](https://github.com/genialis/resolwe-bio/pull/475#discussion_r166314952).

File: docs/CHANGELOG.rst (hunk 1 above)
Comment (hadalin) [reply docs/CHANGELOG.rst]: Fixed.

**Pull request merged**